src/com/infected/model/Player.java

```diff
@@ -138,13 +155,15 @@ public static void resetPlayer() {
     public static void addItem(String item) {
         JsonNode origNode = getPlayerBackPack();
         LinkedHashMap<String, Integer> map = TextParser.jsonNodeToHashMapInt(origNode);
-        if (map.containsKey(item) && Location.getLocationItemsNode().get(Player.getCurrentLocation()).get(item).asInt() >= 1) {
+        System.out.println(Objects.requireNonNull(Location.getLocationItemsNode()).get(Player.getCurrentLocation()));
```

Comment:
is this SOUT meant for debugging or to display to the user?

Reply:
yeah, I'll fix that